android/tutorials/tutorial3/solutions.md

```diff
@@ -72,6 +72,24 @@ class HelloWorldInteractor(
         )
 }
 ```
+
+### HelloWorld
+```kotlin
+interface GreetingsContainer : Rib {
```

Comment:
This is incorrect based on the above.

Reply:
I meant `HelloWorld` class, but for documentation I copied `GreetingsContainer` and forgot to rename the whole class, I'll fix it in a moment